src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/PlaceholderTest.kt

```diff
@@ -0,0 +1,25 @@
+package fi.aalto.cs.apluscourses.e2e
+
+import fi.aalto.cs.apluscourses.e2e.fixtures.*
+import fi.aalto.cs.apluscourses.e2e.steps.CommonSteps
+import fi.aalto.cs.apluscourses.e2e.utils.StepLoggerInitializer
+import fi.aalto.cs.apluscourses.e2e.utils.uiTest
+import org.junit.Assert.*
+import org.junit.Test
+
+class PlaceholderTest {
```

Comment:
Just making sure I understand, is this like a wrapper around the UI tests that wraps it in a normal JUnit test?

Reply:
There are very little "wrapping" actually.

If you mean these lines:
```
fun uiTest(test: RemoteRobot.() -> Unit) {
  RemoteRobot("http://localhost:8082").test()
}
```
then yeah, they kinda "wrap" the method but basically they just create a `RemoteRobot` object and available via `this` keyword.

So basically our UI tests are nothing more than JUnit tests that starts with a RemoteRobot object initialization. The remote robot connects to the (localhosted) server put up by `runIdeForUiTests` and does all the mouse-moving magic. Our JUnit test(s) just call methods of that robot and asserts it returns correct values.